.github/workflows/builder.yml

```diff
@@ -85,6 +85,8 @@ jobs:
       run: |
         echo "Build project"
         make build-kamel bundle-kamelets
+        echo "Preparing the maven overlay"
```

Comment:
I wonder whether we could call `make images` to avoid duplicating the build logic across workflows?

Reply:
Yeah. I think there's some duplication on the actions code and we could probably clean it. My .2 cents: let's open a ticket for this and I'll take care of it on a separate PR. wdyt?

Reply:
Sounds good to me 👍🏼.

Reply:
Deal. I'll take care of the ticket and the PR for this.